commandlineparser/src/Parser.cpp

```diff
@@ -2,7 +2,7 @@
 
 void cmdParser::Parser::AddOptions(std::string short_command,std::string long_command,std::string short_description,std::string long_description) 
 {
-	std::shared_ptr<cmdParser::Options> obj = std::make_shared<cmdParser::Options>(short_command,long_command,short_description,long_description);
+    std::shared_ptr<cmdParser::Options> obj = std::make_shared<cmdParser::Options>(short_command,long_command,short_description,long_description);
```

Comment:
Why this addition and removal of spaces.. every commit indentation is changed ? 
Indentation 
4 spaces
1 tab = 4 spaces